scripts/gen_vimdoc.lua

```diff
@@ -350,12 +350,14 @@ local config = {
     helptag_fmt = function(name)
       if name:lower() == 'treesitter' then
         return 'lua-treesitter-core'
+      elseif name:lower() == 'query' then
+        return 'lua-treesitter-query'
```

Comment:
special-case because currently `treesitter-query` is a different section describing concepts. Maybe it should be moved to a `@brief` in the query.lua module?